app/controllers/api/v1/categories_controller.rb

```diff
@@ -3,6 +3,12 @@
 module Api
   module V1
     class CategoriesController < ApplicationController
+      def index
+        @categories = Category.all
```

Comment:
You need to check if the user is the admin

Reply:
Ok, I added `before_action :admin?, only: :index` to check that the user is the admin